src/main/fc/fc_msp.c

```diff
@@ -1745,6 +1748,9 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
             gyroConfigMutable()->gyro_soft_notch_hz_2 = sbufReadU16(src);
             gyroConfigMutable()->gyro_soft_notch_cutoff_2 = sbufReadU16(src);
         }
+        if (dataSize > 17) {
```

Comment:
Well, how do I know that `dataSize` is deprecated when it is still used all over the place. When something is deprecated then it should be changed by the one that creates the new method. Or am I missing something?